CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -140,6 +170,10 @@ class Row:
     has_content: bool
     num_cells: int
 
+    @property
+    def inflection_cells(self) -> Iterable[InflectionCell]:
+        yield from ()
```

Comment:
Is this one of the things to be implemented in the future?

Reply:
This is the super class that other classes inherit from and overwrite the property; I changed it so that superclass raises `NotImplementedError` so that subclasses can provide a clear implementation!